app/src/main/res/layout/item_join_domain.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.item_join_domain` appears to be unused
 Unused resources make applications larger and slow down builds.